src/setup/index.js

```diff
@@ -35,9 +34,6 @@ export default function () {
             },
             [ONYXKEYS.IS_SIDEBAR_LOADED]: false,
         },
-        registerStorageEventListener: (onStorageEvent) => {
-            listenToStorageEvents(onStorageEvent);
-        },
```

Comment:
Synchronizing storage between tabs is handled internally by Onyx as it depends on the underlying storage implementation. 
See: https://github.com/kidroca/react-native-onyx/blob/b6ad24440fe0b07b314373c1a4cfdf7914febc3f/lib/storage/index.js#L10-L15